pages/osx/open.md

```diff
@@ -6,9 +6,13 @@
 
 `open {{file.ext}}`
 
-- Run a graphical MacOSX application:
+- Run a graphical macOS application:
 
-`open /Applications/{{Application.app}}`
+`open -a {{Application}}`
+
+- Run a graphical macOS app based on the bundle identifier:
```

Comment:
Right, so this brings a dependency on another command. That's why I was doubtful that this would be of use to beginners. And we usually avoid mentioning different commands on a single page if possible.

@sbrl / @waldyrious  - Thoughts ?